docs/configuration.v1alpha1.md

```diff
@@ -180,8 +180,8 @@ spec:
 | agent.deploymentStrategy.canary.replicas |  |
 | agent.deploymentStrategy.canary.validationMode | ValidationMode used to configure how a canary deployment is validated. Possible values are 'auto' (default) and 'manual' |
 | agent.deploymentStrategy.reconcileFrequency | The reconcile frequency of the ExtendDaemonSet. |
-| agent.deploymentStrategy.rollingUpdate.maxParallelPodCreation | The maxium number of pods created in parallel. Default value is 250. |
-| agent.deploymentStrategy.rollingUpdate.maxPodSchedulerFailure | MaxPodSchedulerFailure the maxinum number of not scheduled on its Node due to a scheduler failure: resource constraints. Value can be an absolute number (ex: 5) or a percentage of total number of DaemonSet pods at the start of the update (ex: 10%). Absolute |
+| agent.deploymentStrategy.rollingUpdate.maxParallelPodCreation | The maximum number of pods created in parallel. Default value is 250. |
+| agent.deploymentStrategy.rollingUpdate.maxPodSchedulerFailure | MaxPodSchedulerFailure the maximum number of not scheduled on its Node due to a scheduler failure: resource constraints. Value can be an absolute number (ex: 5) or a percentage of total number of DaemonSet pods at the start of the update (ex: 10%). Absolute |
```

Comment:
Thank you for catching these - this file is generated. Do you mind making the fixes here? https://github.com/DataDog/datadog-operator/blob/58df38cd757b469a5ac4d324e0d0032cadf30b18/apis/datadoghq/v1alpha1/datadogagent_types.go#L283-L287

Reply:
Totally makes sense, was not aware it was generated. Updated.